pty_windows.go

```diff
@@ -0,0 +1,160 @@
+package pty
+
+import (
+	"os"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+var (
+	kernel32DLL *syscall.DLL
+
+	createPseudoConsole        *syscall.Proc
+	resizePseudoConsole        *syscall.Proc
+	closePseudoConsole         *syscall.Proc
+	getConsoleScreenBufferInfo *syscall.Proc
+
+	initializeProcThreadAttributeList *syscall.Proc
+	updateProcThreadAttribute         *syscall.Proc
+	deleteProcThreadAttributeList     *syscall.Proc
+)
+
+func init() {
+	var err error
+	kernel32DLL, err = syscall.LoadDLL("kernel32.dll")
+	if err != nil {
+		panic(err)
+	}
+
+	createPseudoConsole = kernel32DLL.MustFindProc("CreatePseudoConsole")
+	resizePseudoConsole = kernel32DLL.MustFindProc("ResizePseudoConsole")
+	closePseudoConsole = kernel32DLL.MustFindProc("ClosePseudoConsole")
+	getConsoleScreenBufferInfo = kernel32DLL.MustFindProc("GetConsoleScreenBufferInfo")
+
+	initializeProcThreadAttributeList = kernel32DLL.MustFindProc("InitializeProcThreadAttributeList")
+	updateProcThreadAttribute = kernel32DLL.MustFindProc("UpdateProcThreadAttribute")
+	deleteProcThreadAttributeList = kernel32DLL.MustFindProc("DeleteProcThreadAttributeList")
```

Comment:
none of those calls can fail or return nil? Could you add a small comment about it and/or add a check nil check to have a clean error instead of a panic later on during the runtime.

Reply:
same as https://github.com/creack/pty/pull/109#discussion_r607197693

MustFindProc will panic if error happened when calling.